src/main/java/nl/lucasridder/lhub/LHub.java

```diff
@@ -73,8 +73,6 @@ public void run() {
     //On disable
     @Override
     public void onDisable() {
-        //stop plugin
-        saveConfig();
```

Comment:
I don't understand why you chose to remove this call, it's always a good idea to save the config when disabling, just in case `ConfigManager.set` was used somewhere instead of `ConfigManager.setAndSave`.